stories/profile/LessonsChallenges.stories.tsx

```diff
@@ -0,0 +1,173 @@
+import * as React from 'react'
+import LessonsChallenges, {
+  ChallengeStatus
+} from '../../components/LessonsChallenges'
+
+export default {
+  component: LessonsChallenges,
+  title: 'Components/LessonChallenges'
+}
+
+const completedChallenges = [
+  { challengeNumber: 1, challengeStatus: 'passed' },
+  { challengeNumber: 2, challengeStatus: 'passed' },
+  { challengeNumber: 3, challengeStatus: 'passed' },
+  { challengeNumber: 4, challengeStatus: 'passed' },
+  { challengeNumber: 5, challengeStatus: 'passed' },
+  { challengeNumber: 6, challengeStatus: 'passed' },
+  { challengeNumber: 7, challengeStatus: 'passed' },
+  { challengeNumber: 8, challengeStatus: 'passed' },
+  { challengeNumber: 9, challengeStatus: 'passed' },
+  { challengeNumber: 10, challengeStatus: 'passed' }
+]
+
+const defaultChallenges = [
+  { challengeNumber: 1, challengeStatus: 'default' },
+  { challengeNumber: 2, challengeStatus: 'default' },
+  { challengeNumber: 3, challengeStatus: 'default' },
+  { challengeNumber: 4, challengeStatus: 'default' },
+  { challengeNumber: 5, challengeStatus: 'default' },
+  { challengeNumber: 6, challengeStatus: 'default' },
```

Comment:
could you make some of these challengeStatus null? or some does not even have this key?

Reply:
understood